lib/vanity/metric/active_record.rb

```diff
@@ -99,7 +101,13 @@ def last_update_at
       def after_create(record)
         return unless @playground.collecting?
         count = @ar_column ? (record.send(@ar_column) || 0) : 1
-        call_hooks record.send(@ar_timestamp), nil, [count] if count > 0 && @ar_scoped.exists?(record.id)
+
+        identity = Vanity.context.vanity_identity rescue nil
+        identity ||= if @ar_identity_block
+          @ar_identity_block.call( record )
```

Comment:
You can tell I have a different coding style in my day-to-day work. :)  Fixes incoming.